### DISCUSSION TITLE
51统计代码更换

### DISCUSSION THREAD

**Discussion by cao-gift** [imsyy/home#17] [General]
看到JS文件夹里有51统计JS文件，怎么更换成自己的？小白勿喷

**Comment by imsyy** [comment]
![image](https://user-images.githubusercontent.com/42232682/184662485-96857bb8-c0be-4b23-90cc-ba712a676cc8.png)
登陆你的 [51统计后台](https://v6.51.la/user/application) 添加站点后，用这里面的代码替换了就行了

**Comment by musnows** [comment]
直接在`index.html`的head里面添加自己的应该就行了吧

**Reply by imsyy** [reply]
> 直接在`index.html`的head里面添加自己的应该就行了吧

复制到 `index.html` 页面的 `<head />` 前就行，我重构把这个给搞忘了 😂
